velox/docs/functions/presto/binary.rst

```diff
@@ -62,6 +62,10 @@ Binary Functions
 
     Encodes ``binary`` into a base64 string representation using the `URL safe alphabet <https://www.rfc-editor.org/rfc/rfc4648#section-5>`_.
 
+.. function:: from_base64url(string) -> varbinary
```

Comment:
Can we move this entry to preserve the alphabetical order?

Reply:
@Yuhta Absolutely! done with changes, thanks for all your valuable feedback, highly appreciate it!